### ISSUE TITLE
Better Access to Card Names

### ISSUE THREAD

**Issue by ichabod801** [ichabod801/t_games#327]
Currently CrazyEights is doing this:

```python
self.card_class.rank_names[self.card_class.ranks.index(self.change_rank)]
```

There should be a dict or something to get that information easier.

**Comment by ichabod801** [comment]
This is a child of issue #413: update cards.py.

**Comment by ichabod801** [comment]
This was implemented as part of issue #414. It is now:

```python
self.deck.rank_set.names[self.change_rank]
```

It still has four dot operators, but otherwise it's a straightforward dictionary lookup.
